open-api/rest-catalog-open-api.yaml

```diff
@@ -3952,121 +4018,102 @@ components:
             type: integer
           description: "List of equality field IDs"
 
-    PlanTableRequest:
+    PlanTableScanRequest:
       type: object
       properties:
         snapshot-id:
           description:
-            The ID of the snapshot to use for the table scan.
+            Identifier for the snapshot to scan in a point-in-time scan
```

Comment:
```suggestion
            Identifier of a snapshot to be utilized in a point-in-time scan
```

Reply:
I think it is better to use active instead of passive voice. The verb "scan" is more clear than "to be utilized" right?

Reply:
Thats fair, I should have also marked this as a [nit] as i saw we had the word "scan" used twice in the sentence. Im ok with the original as well.